### PR TITLE
Added [KB] for snyk/action

### PR DESCRIPTION
**Pull request by Devils-Knight**

### CODE REVIEW

File: knowledge-base/snyk/action/action-security.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: 'Snyk-dokker action'
```
Comment (varunsh-coder) [inline knowledge-base/snyk/action/action-security.yml:1]: @Devils-Knight github.com/snyk/action returns a 404 page not found. Can you please change the file name to `snyk/actions`?

File: knowledge-base/snyk/actions/action-security.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: 'Snyk-Docker action'
```
Comment (varunsh-coder) [inline knowledge-base/snyk/actions/action-security.yml:1]: Name should be `Snyk Docker` as per https://github.com/snyk/actions/blob/master/docker/action.yml

File: knowledge-base/snyk/actions/action-security.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+outbound-endpoints:
+  - fqdn: snyk.io
```
Comment (varunsh-coder) [inline knowledge-base/snyk/actions/action-security.yml:3]: Please run the workflow to find the outbound endpoints.

**Comment by Devils-Knight** [comment]
No other outbounds were found using snyk API token in the workflow, <a href="https://app.stepsecurity.io/github/Devils-Knight/Knowledge-base/actions/runs/1696422302" > harden-runner-link </a> of snyk/action.

File: knowledge-base/snyk/actions/action-security.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+name: 'Snyk Docker'
```
Comment (varunsh-coder) [inline knowledge-base/snyk/actions/action-security.yml:1]: @Devils-Knight please update the file path to `snyk/actions/docker/action-security.yml`

**Pull request merged**